community-roles.md

```diff
@@ -1,49 +1,46 @@
-- [Repository Manager](#repository-manager)
 - [Community Manager](#community-manager)
+- [Repository Manager](#repository-manager)
+- [Project Release Manager](#project-release-manager)
 - [Solid Core Contributor](#solid-core-contributor)
 - [Project Core Contributor](#project-core-contributor)
-- [Project Contributor](#project-contributor)
-- [Project Release Manager](#project-release-manager)
 
 ## Community Leader
-The Community Leader defines the governing vision of the Solid project, and
-channels that vision into the projects and initiatives created to realize it.
-They are responsible for selecting core contributors, approving official
-projects and initiatives, prioritizing official project roadmaps, and have the
-final say on project disputes.
-
-## Repository Manager
-The Repository Manager controls Github, NPM, and other Solid organization
-properties related to the storage, management, and distribution of official
-Solid projects. They are responsible for keeping these properties clean and
-well-organized on the community's behalf.
+The Community Leader defines the governing vision of the Solid project, and channels that vision into the projects and initiatives created to realise it. They are responsible for selecting core contributors, approving official projects and initiatives, prioritising official project roadmaps, and have the final say on project disagreements. The Community Leader has admin rights of Solid tools and can optionally attend the weekly recurring community support meeting.
 
 ## Community Manager
-Community Managers are responsible for community outreach, ensuring that
-community members are able to realize the full value of solid, and have what
-they need to contribute value back if they choose to do so.
+The Community Manager is responsible for community outreach to facilitate contributions and implementation of the governance of Solid. The Community Manager is responsible for merging pull requsets concerning the community repository and processing and implementing suggestions around Solid governance. The Community Manager has admin rights of Solid tools and must moderate the weekly recurring community support meeting. 
+
+## Repositories Manager
+The Repositories Manager controls Github, NPM, and other Solid organisation properties related to the storage, management, and distribution of official Solid projects. They are responsible for keeping these properties clean and well-organised on behalf of the Solid community. The Repositories Manager has admin rights of Solid tools must attend the weekly recurring community support meeting.
 
 ## Solid Core Contributor
-Solid Core Contributors work across projects on initiatives that enrich the
-Solid ecosystem as a whole. They must attend regular stand-ups and strategy
-meetings.
+Solid Core Contributors work across projects on initiatives that enrich the Solid ecosystem as a whole. Solid Core Contributors must attend the weekly recurring community support meeting.
 
```

Comment:
I have to disagree with removing a requirement that core contributors attend regular stand-ups and strategy meetings. IMO - the community support meeting is one such regular strategy meeting, but there's a lot more to it than only that one meeting. I think the original language includes the community support meeting without removing other criteria that's ultimately important elements of core contribution.